src/main/webapp/i18n/de/plagiarism.json

```
@@ -63,6 +64,7 @@
                     "plagiarism": "Unterschleid",
                     "warning": "Warnung",
                     "pointDeduction": "Punktabzug",
+                    "noPlagiarism": "Kein Unterschleid",
```

```suggestion
                    "noPlagiarism": "Kein Plagiat",
```
If you want to keep the distinction between "Plagiat" and "Unterschleif", you have to fix the spelling (last character)